### DISCUSSION TITLE
What to .gitignore?

### DISCUSSION THREAD

**Discussion by EvgeniiaVak** [nhost/nhost#2243] [Q&A] [answered]
I went through the https://docs.nhost.io/cli/local-development#initialize-local-project section and it created more than a thousand new files, and only `.secrets.nhost` is added automatically to gitignore. Here is the overview of the new files:
<img width="245" alt="Screenshot 2023-09-16 at 16 08 50" src="https://github.com/nhost/nhost/assets/27793901/b1875bb2-dc4e-47f0-9bc5-5edbd1316d5e">

I'm guessing `.nhost` and `.secrets` should also be ignored, and `functions` and `nhost` shouldn't, is that correct?

**Comment by dbarrosop** [comment, marked as answer]
That's actually a bug. The contents should've been:

```
.secrets
.nhost
```

Thanks for reporting, will fix (moving the convo to #2244)

> I'm guessing .nhost and .secrets should also be ignored, and functions and nhost shouldn't, is that correct?

that is correct
